export function handleError(err) {
    console.log(err);
    throw err;
};
